packet_filter_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Jigsaw Operations LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tests for the packet_filter library.
+"""
+
+
+import io
+
+import packet_filter
+import pytest
+
+
+@pytest.fixture()
+def badips_map():
+  return {
+    '1.1.1.1': 'Source A',
+    '4.5.6.7': 'Best IOCs',
+    '100.12.34.56': 'Threatbusters',
+  }
+
+class MockLogging:
```

Comment:
No need to mock logs with the OO approach I mention above.

Reply:
Yeah, I can remove the logging mock, it was more useful before I refactored the code to return a boolean.

An OO approach might still need some mocks but not on the logger, and I agree it would be unnecessary after separating detection and logging logic.

I've removed the mock class here.